core/login.go

```diff
@@ -17,15 +17,18 @@ package core
 
 import (
 	"fmt"
-	"net/url"
 	. "github.com/vmware/priam/util"
+	"net/url"
 )
 
+/* Local user domain as defined in IDM (might change in the future when multiple local domains will be supported) . */
+const LOCAL_USER_DOMAIN = "Local Users"
```

Comment:
In idiomatic go constants are preferred to be camel case: http://stackoverflow.com/questions/22688906/go-naming-conventions-for-const . I'd prefer not having the LocalUsers constant at all (esp since I think "Local Users" can be renamed) or at least not public, but since we need to rearrange cli login soon anyway, it's fine.